web/views/page_view.ex

```diff
@@ -1,3 +1,31 @@
 defmodule Tudo.PageView do
   use Tudo.Web, :view
+  use Rummage.Phoenix.View
+
+  def generate_avatar(issue) do
+    issue.assignees
+    |> Enum.map(fn assignee -> assignee |> String.split(";") |> Enum.at(0) end)
+  end
+
+  def get_label_data(issue) do
+    issue.labels
+    |> Enum.map(fn label ->
+      label
+      |> String.split(";")
+      |> List.to_tuple
+      |> get_label_font_color
+    end)
+  end
+
+  def get_label_font_color({label_hex, label_name}) do
+    rgb_value = label_hex
+    |> String.replace("#", "")
+    |> String.to_integer(16)
+
+    case rgb_value > 0xffffff / 2 do
+      true -> %{label_hex: label_hex, label_name: label_name, font_hex: '#000'}
+      false -> %{label_hex: label_hex, label_name: label_name, font_hex: '#fff'}
+    end
```

Comment:
Be weary of using single quotes in elixir, they are interpreted differently and may not work with the `String` module.
(e.g typing this into iex: `is_list('string') # outputs true`)

You could have written this function like this to be stricter and more concise:
```elixir
def get_label_font_color({"#" <> hex_code = label_hex, label_name}) do
  %{label_hex: label_hex,
    label_name: label_name,
    font_hex: if String.to_integer(hex_code, 16) > 0xffffff / 2 do "#000" else "#fff" end}
end
```